src/main/java/ch/njol/skript/expressions/ExprTargetedBlock.java

```diff
@@ -36,36 +36,47 @@
 import org.eclipse.jdt.annotation.Nullable;
 
 @Name("Targeted Block")
-@Description("The block at the crosshair. This regards all blocks that are not air as fully solid, e.g. torches will be like a solid stone block for this expression.")
-@Examples({"# A command to set the block a player looks at to a specific type:",
-		"command /setblock &lt;material&gt;:",
-		"\ttrigger:",
-		"\t\tset targeted block to argument"})
+@Description({"The block at the crosshair. This regards all blocks that are not air as fully solid, e.g. torches will be like a solid stone block for this expression.",
+	"The actual target block will regard the actual hit box of the block."})
+@Examples({
+	"set target block of player to stone",
+	"set target block of player to oak_stairs[waterlogged=true]",
+	"break target block of player using player's tool",
+	"give player 1 of type of target block",
+	"teleport player to location above target block",
+	"kill all entities in radius 3 around target block of player",
+	"set {_block} to actual target block of player",
+	"break actual target block of player"
+})
 @Since("1.0")
 public class ExprTargetedBlock extends PropertyExpression<Player, Block> {
 
 	static {
 		Skript.registerExpression(ExprTargetedBlock.class, Block.class, ExpressionType.COMBINED,
-				"[the] target[ed] block[s] [of %players%]", "%players%'[s] target[ed] block[s]",
-				"[the] actual[ly] target[ed] block[s] [of %players%]", "%players%'[s] actual[ly] target[ed] block[s]");
+			"[the] target[ed] block[s] [of %players%]", "%players%'[s] target[ed] block[s]",
+			"[the] actual[ly] target[ed] block[s] [of %players%]", "%players%'[s] actual[ly] target[ed] block[s]");
```

Comment:
Can 'exact' be added to the pattern to reflect the Spigot method?

Reply:
I thought about that but the pattern is already such a god damn mess I didn't want to make it worse